docs/docs/supply-chain/vex.md

```diff
@@ -470,6 +474,100 @@ does not match:
 - `pkg:maven/com.google.guava/guava@24.1.1?classifier=sources`
     - `classifier` must have the same value.
 
+Here is a proposed addition to the Trivy VEX documentation, focusing on how VEX is applied to dependency graphs:
```

Comment:
I suppose this is here by mistake

Reply:
AI added 😄 Fixed in https://github.com/aquasecurity/trivy/pull/6864/commits/1fac6335eb8f56e9fdf2fb911b649333ee4bef60